src/main/scala/uk/gov/nationalarchives/tdr/api/auth/ValidationTag.scala

```diff
@@ -0,0 +1,14 @@
+package uk.gov.nationalarchives.tdr.api.auth
```

Comment:
It would be good to move this out of the auth package if it's being used for more than just auth.

Reply:
Moved to graphql package as separately discussed